.gitignore

```diff
@@ -2,6 +2,8 @@
 .DS_Store
 AxonDeepSeg\.egg-info/
 
+bin/
```

Comment:
`/bin` would be more accurate actually, `bin/` can match `bin` directories at any depth

Reply:
6d24bcf

Reply:
7d4795e